airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -289,6 +289,13 @@
                 ],
                 "default": "After loading Data in the destination",
                 "order": 7
+              },
+              "heartbeat_action_query": {
```

Comment:
Checking: because this field is optional, existing connection lacking it would just keep working
i.e non-breaking change

Reply:
Yes - this property is only set if it's populated, otherwise I've verified that the behavior is the same. The default value is empty